spring-integration-core/src/test/java/org/springframework/integration/config/AggregatorParserTests.java

```diff
@@ -266,5 +302,19 @@ public void testAggregatorFailureIfMutuallyExclusivityPresent() {
 		return MessageBuilder.withPayload(payload).setCorrelationId(correlationId).setSequenceSize(sequenceSize)
 				.setSequenceNumber(sequenceNumber).setReplyChannel(outputChannel).build();
 	}
-	
+
+	public static class MyMGP extends SimpleMessageGroupProcessor {
```

Comment:
MC is a little different - there is no transformation.

In this case, we're changing a message group to its collection of messages.

Subtle, perhaps, but different.